oracle/src/sender.js

```diff
@@ -123,53 +123,44 @@ async function main({ msg, ackMsg, nackMsg, channel, scheduleForRetry, scheduleT
       }
 
       try {
-        let txNonce
         if (isResend) {
           const tx = await web3Instance.eth.getTransaction(job.txHash)
 
-          if (tx === null) {
-            logger.info(`Transaction ${job.txHash} was not found, dropping it`)
-            return
-          }
-          if (tx.blockNumber !== null) {
+          if (tx && tx.blockNumber !== null) {
             logger.info(`Transaction ${job.txHash} was successfully mined`)
```

Comment:
```suggestion
            logger.debug(`Transaction ${job.txHash} was successfully mined`)
```

Reply:
Since we optimistically assume that most transactions will be mined, so, this will overload the logs.